### PR TITLE
Add a set_run_classes method to MLflow extension, to store the classe…

### PR DESCRIPTION
**Pull request by lpenet**
Companion backend PR: https://github.com/dataiku/dip/pull/15556

### CODE REVIEW

**Review by alarive — APPROVED** [review]
LGTM+tested

File: dataikuapi/dss/mlflow.py [hunk 1]
```diff
@@ -131,3 +133,68 @@ def clean_experiment_tracking_db(self):
         This call requires an API key with admin rights
         """
         self.client._perform_raw("DELETE", "/api/2.0/mlflow/extension/clean-db/%s" % self.project_key)
+
+    def set_run_inference_info(self, run_id, prediction_type, classes, code_env_name, target):
+        """
+        Sets the prediction_type of the model, and the classes, if it is a classification model.
+
+        prediction_type must be one of:
+        - NON_TABULAR if the model is not tabular
```
[the remark this answers is not in the document]
Comment (lpenet) [reply dataikuapi/dss/mlflow.py:142]: So, we shall use `OTHER`.

File: dataikuapi/dss/mlflow.py [hunk 2]
```diff
@@ -131,3 +133,68 @@ def clean_experiment_tracking_db(self):
...
+
+        if classes and prediction_type not in {"BINARY_CLASSIFICATION", "MULTICLASS"}:
+            raise ValueError('Classes can be specified only for BINARY_CLASSIFICATION or MULTICLASS prediction types')
+        if prediction_type in {"BINARY_CLASSIFICATION", "MULTICLASS"}:
+            if not classes:
+                raise ValueError('Classes must be specified for {} prediction type'.format(prediction_type))
+            if not isinstance(classes, list):
+                raise ValueError('Wrong type for classes: {}'.format(type(classes)))
+            for cur_class in classes:
+                if not cur_class:
```
[the remark this answers is not in the document]
Comment (lpenet) [reply dataikuapi/dss/mlflow.py]: :facepalm: Thanks

**Review by vthorey — APPROVED** [review]
Tested and LGTM

**Pull request merged**